src/lib/machineAdjustments.ts

```diff
@@ -0,0 +1,346 @@
+interface AdjustmentPosition {
+  name: string
+  range: string[]
+  translation?: string
+}
+
+interface MachineAdjustmentTemplate {
+  adjustments: AdjustmentPosition[]
+  primaryAdjustmentIndex?: number
+}
+
+export class MachineAdjustmentsRecord {
+  private _machineAdjustmentTemplate: MachineAdjustmentTemplate
+
+  constructor (machineAdjustmentTemplate: MachineAdjustmentTemplate) {
+    this._machineAdjustmentTemplate = machineAdjustmentTemplate
+  }
+
+  protected setMachineAdjustmentTemplate (machineAdjustmentTemplate: MachineAdjustmentTemplate) {
+    this._machineAdjustmentTemplate = machineAdjustmentTemplate
+  }
+
+  get adjustments () {
+    return this._machineAdjustmentTemplate.adjustments
+  }
+
+  get primaryAdjustmentIndex () {
+    return this._machineAdjustmentTemplate.primaryAdjustmentIndex
+  }
+}
+
+export const machineAdjustments = {
```

Comment:
I thought I left this in the last review, but I must have not added it.  

This structure makes sense.  But it seems a little complicated.  Because adjustments are made up of `seat`, `leftPosition`, `rightPosition`, `start`,  and `stop`.  Wouldn't it be better to define the following?

```
'001289': {
    'seat': {
        'range': ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 'A', 'C', 'E', 'F', 'H', 'J'],
        'alias': 'Back',
        'required': false
    },
    'start': {
        'range': ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 'A'],
        'alias': 'Start',
        'required': true
    },
    'stop': {
        'range':  ['1', '2', '3', '4', '5', '6', '7', '8', '9', '0', 'A', 'C', 'E'],
        'alias': 'Stop',
        'required': false
    },
    'leftPosition': {
        'range':  ['1', '2', '3', '4', '5', '6', '7', '8'],
        'alias': 'Leg',
        'required': false
    }
}
```